docs/index.rst

```diff
@@ -57,6 +57,15 @@ Pillow for enterprise is available via the Tidelift Subscription. `Learn more <h
    :target: https://pypi.org/project/Pillow/
    :alt: Number of PyPI downloads
 
+Overview
+========
+
+The Python Imaging Library adds image processing capabilities to your Python interpreter.
+
+This library provides extensive file format support, an efficient internal representation, and fairly powerful image processing capabilities.
+
+The core image library is designed for fast access to data stored in a few basic pixel formats. It should provide a solid foundation for a general image processing tool.
+
```

Comment:
I'd move this up, right below the title. Here users need to scroll a lot to find it.

Reply:
If others have thoughts, feel free to disagree with me, but I don't see the need for this
- for ReadTheDocs, if you're on a desktop, you don't need to scroll to see this new text in its current position
- the current first line in both ReadTheDocs and the README includes 'Python Imaging Library'. I think those three words convey the idea of this repository fairly well. If a visitor is interested after that, they are interested enough to scroll further. If they're not, then I don't think the 'Overview' is going to convince them otherwise.

Reply:
Yep, I think this PR is an improvement. Thanks both!